src/app/server/Server.cpp

```diff
@@ -389,23 +271,17 @@ CHIP_ERROR OpenDefaultPairingWindow(ResetFabrics resetFabrics, uint16_t commissi
 
     if (resetFabrics == ResetFabrics::kYes)
     {
-        EraseAllFabricsUpTo(gNextAvailableFabricIndex);
-        EraseAllSessionsUpTo(gSessionIDAllocator.Peek());
+        gFabrics.DeleteAllFabrics();
```

Comment:
This resetFabrics is argument seems to do something unexpected. Why is this set to kYes, and why?

Reply:
This was mainly needed for ESP32 (m5stack) application, as we have a UI to clear the settings on the device.
It got reused for remove all fabrics command, that is no longer supported by the cluster.
We can eventually remove it, but removing it here will cause some behavior change for m5stack application.

Reply:
I don't mind at all that we would have behavior changes to non-spec-compliant sample applications. It's more important that we reach spec compliance, especially if doing so is "low-touch". It's not a large fundamental breakage 
here in that sample. Just a slight "manual testing usability" concern